### ISSUE TITLE
Issue with NEURON mods of model id 472450023

### ISSUE THREAD

**Issue by anujanegi** [OpenSourceBrain/AllenInstituteNeuroML#33]
For [472450023](http://celltypes.brain-map.org/mouse/experiment/electrophysiology/464198958), when simulating the cell using the downloaded data, this is thrown out.

 ```
NEURON configured
NEURON: Im is not a MECHANISM
 near line 1
 forsec "soma" { insert Im }
                         ^
NEURON: Ih is not a MECHANISM
 near line 1
 forsec "soma" { insert Ih }
                         ^
NEURON: NaTs is not a MECHANISM
 near line 1
 forsec "soma" { insert NaTs }
                           ^
NEURON: Nap is not a MECHANISM
 near line 1
 forsec "soma" { insert Nap }
                          ^
NEURON: K_P is not a MECHANISM
 near line 1
 forsec "soma" { insert K_P }
                          ^
NEURON: K_T is not a MECHANISM
 near line 1
 forsec "soma" { insert K_T }
                          ^
NEURON: SK is not a MECHANISM
 near line 1
 forsec "soma" { insert SK }
                         ^
NEURON: Kv3_1 is not a MECHANISM
 near line 1
 forsec "soma" { insert Kv3_1 }
                            ^
NEURON: Ca_HVA is not a MECHANISM
 near line 1
 forsec "soma" { insert Ca_HVA }
                             ^
NEURON: Ca_LVA is not a MECHANISM
 near line 1
 forsec "soma" { insert Ca_LVA }
                             ^
NEURON: CaDynamics is not a MECHANISM
 near line 1
 forsec "soma" { insert CaDynamics }
                                 ^
NEURON: CaDynamics is not a MECHANISM
 near line 1
 forsec "soma" { insert CaDynamics }
                                 ^
k_ion mechanism not inserted in section soma[0]
NEURON: 
 near line 1
 forsec "soma" { ek = -107 }
                            ^
na_ion mechanism not inserted in section soma[0]
NEURON: 
 near line 1
 forsec "soma" { ena = 53 }
```

I think this is what caused the weird NEURON figure for this model ([here](https://github.com/OpenSourceBrain/AllenInstituteNeuroML/blob/c480fddee45053720f28b833c3d427b6272954f3/CellTypesDatabase/models/README.md)).

For sweep 29, this is the current LEMS simulation, which looks right compared to the fig [here](http://celltypes.brain-map.org/mouse/experiment/electrophysiology/464198958):
![LEMS_472450023](https://user-images.githubusercontent.com/29161453/182174904-d13f6bb2-6841-4b86-a7c9-1a5153aa1d08.png)

@pgleeson

**Comment by pgleeson** [comment]
@anujanegi I believ this can be solved by going into the 472450023 directory and running: nrnivmodl modfiles

After that running: python run_one.py -test_perisomatic3 (as in my latest commit), worked fine.

**Comment by anujanegi** [comment]
My bad. Updated the README in #31.
